### PR TITLE
INTGR-271: Update links and titles

### PR DESCRIPTION
**Pull request by tranjennie**

### CODE REVIEW

File: src/pages/myAdmin/guides/concepts.tsx [hunk 1]
```diff
@@ -348,8 +348,7 @@ export default function Concepts() {
                         JSON
                     </a>{" "}
                     format. The{" "}
-                    {/* TODO: update link */}
-                    <Link to=""> Reference</Link>{" "}
+                    <Link to="./../../apiReference/methods"> Reference</Link>{" "}
```
Comment (kmets89) [inline src/pages/myAdmin/guides/concepts.tsx:351]: not the right path

File: src/pages/myAdmin/guides/gettingStarted.tsx [hunk 2]
```diff
@@ -19,22 +18,21 @@ const accessApi: ReactNode = (
...
+        <h2>Test environment</h2>
         <p>
             A sandbox environment is a testing environment that is available to developers who wish to test their application before making API calls to the live system. The test environment can be accessed at:
         </p>
         <code className="small-code-sample">https://myadminapitest.geotab.com/v2/MyAdminApi.ashx</code>
         <p>
             A few methods are not supported in the test environment and will throw exceptions when called. For more information, see the{" "}
-            {/* TODO: update link */}
-            <Link to="">Reference</Link>{" "}
+            <Link to="./../../apiReference/methods">Reference</Link>{" "}
```
Comment (kmets89) [inline src/pages/myAdmin/guides/gettingStarted.tsx:28]: also this one

File: src/pages/myAdmin/guides/gettingStarted.tsx [hunk 3]
```diff
@@ -46,14 +44,11 @@ const authenticateApi: ReactNode = (
         <code className="small-code-sample">{`{"result":{"userId":"x12345x2-172x-4d04-8xx2-xx9e088c5xxx","sessionId":"cff4e88b-931b-4363-ae4f-35b5ed169133","lastLogonDate":"2013-11-04T15:01:00.000Z","roles":[{"comments":"Third Party Integrator role","name":"Third-Party-Integrator"}],"name":"user@geotab.com"}}`}</code>
         <p>
             The API key is found in the <code className="small-code-sample">userId</code> property and the session ID is found in the <code className="small-code-sample">sessionId</code> property. Both of these properties must be included on all other API calls. If the session ID has expired, the API will return a <code className="small-code-sample">SessionExpiredException</code> which requires a call to Authenticate to get the new session ID. For more information, see the{" "}
-            {/* TODO: update link */}
-            <Link to="">Reference</Link>{" "}
+            <Link to="./../../apiReference/methods">Reference</Link>{" "}
```
Comment (kmets89) [inline src/pages/myAdmin/guides/gettingStarted.tsx:47]: for all these, please copy the path from App.tsx

**Review by kmets89 — CHANGES_REQUESTED** [review]
please review all links and follow the pathing from our routes in App.tsx, not the old file paths from the current repo

File: src/pages/myAdmin/guides/gettingStarted.tsx (hunk 3 above)
Comment (tranjennie) [reply src/pages/myAdmin/guides/gettingStarted.tsx]: Has been updated to "/myAdmin/apiReference/methods"

File: src/pages/myAdmin/guides/concepts.tsx (hunk 1 above)
Comment (tranjennie) [reply src/pages/myAdmin/guides/concepts.tsx]: Has been updated to "/myAdmin/apiReference/methods"

File: src/pages/myAdmin/guides/gettingStarted.tsx (hunk 2 above)
Comment (tranjennie) [reply src/pages/myAdmin/guides/gettingStarted.tsx]: Has been updated to "/myAdmin/apiReference/methods"

**Comment by tranjennie** [comment]
Hey @kmets89 ! I updated the links to "/myAdmin/apiReference/methods". That page is not available yet but is being implemented on INTGR-255

File: src/pages/myAdmin/guides/concepts.tsx
Comment (kmets89) [inline src/pages/myAdmin/guides/concepts.tsx:1]: on line 31, there's a CodeSample tag inside a p tag, which throws an invalid HTML error, please move it outside

File: src/pages/myGeotab/addIns/addInStorage.tsx
Comment (kmets89) [inline src/pages/myGeotab/addIns/addInStorage.tsx:1]: these links are still not correct, if you don't want to guess what they'll be please add TODOs here

File: src/pages/myGeotab/codeSamples/javascriptSamples.tsx
Comment (kmets89) [inline src/pages/myGeotab/codeSamples/javascriptSamples.tsx:1]: I know you copied this from the current site but Joseph and I discussed some weirdness on this page we'd like to change
Can you also lowercase drivers and zones on lines 13 and 37

File: src/pages/myGeotab/codeSamples/javascriptSamples.tsx [hunk 4]
```diff
@@ -467,63 +467,63 @@ const pageSections: TableOfContentsItem[] = [
 
     {
         "elementId": "javascript-example-add-driver",
-        "summary": "Add Driver",
+        "summary": "Add driver",
```
Comment (kmets89) [inline src/pages/myGeotab/codeSamples/javascriptSamples.tsx:470]:
```suggestion
        "summary": "Add a driver",
```

File: src/pages/myGeotab/codeSamples/javascriptSamples.tsx [hunk 5]
```diff
@@ -467,63 +467,63 @@ const pageSections: TableOfContentsItem[] = [
...
         "details": addDriver
     }, {
         "elementId": "javascript-example-add-zone",
-        "summary": "Add Zone",
+        "summary": "Add zone",
```
Comment (kmets89) [inline src/pages/myGeotab/codeSamples/javascriptSamples.tsx:474]:
```suggestion
        "summary": "Add a zone",
```

File: src/pages/myGeotab/codeSamples/javascriptSamples.tsx [hunk 6]
```diff
@@ -467,63 +467,63 @@ const pageSections: TableOfContentsItem[] = [
...
         "details": dataFeed
     }, {
         "elementId": "javascript-example-display-odometer",
-        "summary": "Display Odometer",
+        "summary": "Display odometer",
         "details": displayOdometer
     }, {
         "elementId": "javascript-example-get-count",
-        "summary": "Get Count",
+        "summary": "Get count",
```
Comment (kmets89) [inline src/pages/myGeotab/codeSamples/javascriptSamples.tsx:486]:
```suggestion
        "summary": "Get a count of vehicles and users",
```

**Review by kmets89 — CHANGES_REQUESTED** [review]

**Review by kmets89 — APPROVED** [review]

**Pull request merged**